CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- Major: Add Barbarian Assault Gambling notifications. (#150)
```

Comment:
```suggestion
- Major: Add Barbarian Assault high-level Gambling notifications. (#150)
```

Throw this nitpick in since I wrote this Changelog while thinking about the possible low&medium implementation